resultmap.go

```diff
@@ -54,41 +54,46 @@ func (rm *ResultMap) Len() int {
 	return l
 }
 
-// FilteredCheckMap represents a concurrent-safe map to store CheckResults generated by concurrent jobs.
-type FilteredCheckMap struct {
+type FilteredResult struct {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported type FilteredResult should have comment or be unexported